main.go

```diff
@@ -213,6 +264,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Only start the controller when the leader election is won
+	mgr.Add(manager.RunnableFunc(func(stop <-chan struct{}) error {
```

Comment:
nit: ideally we should startController here, and explicitly wait for cache sync before call mgr.start.  See https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/main.go#L154

The benefit is the controller's will not operator until the cache is synced.
In current code which relies on mgr, controllers might work without waiting for the pod cache, and only blocks on when it need pod information.